accelerator/main.c

```diff
@@ -21,9 +21,15 @@ int main(int argc, char* argv[]) {
   }
 
   // Initialize logger
+#ifdef DEBUG
   if (logger_helper_init(LOGGER_DEBUG) != RC_OK) {
     return EXIT_FAILURE;
   }
+#else
+  if (logger_helper_init(LOGGER_INFO) != RC_OK) {
+    return EXIT_FAILURE;
+  }
+#endif
```

Comment:
Can these wrapped into another function such as `ta_logger_init`?
Then we can reuse it in different files.

Reply:
Sure! I am going to do that.